src/components/RelicScorerTab.jsx

```diff
@@ -81,6 +95,8 @@ export default function RelicScorerTab() {
     setScorerId(id)
     SaveState.save()
 
+    window.history.pushState({ id: id }, `profile: ${id}`, PageToRoute[AppPages.RELIC_SCORER] + `/?uid=${id}`)
```

Comment:
window.history.pushState has the annoying side effect of duplicating this same page in the history navigation so the user cant go back to their previous page, I think replaceState is probably what we want